release-notes/3.0/preview/3.0.0-preview-known-issues.md

```diff
@@ -180,7 +180,7 @@ When building or running the react template with authentication, after opening t
       }
       ```
 
-**Runtime compilation using Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation throws an error stating "Cannot find reference assembly"**:
+- **Runtime compilation using Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation throws an error stating "Cannot find reference assembly"**:
```

Comment:
All the content should be indented under this entry

Reply:
🆙 📅